pkgs/top-level/all-packages.nix

```diff
@@ -2774,6 +2774,8 @@ let
 
   pytrainer = callPackage ../applications/misc/pytrainer { };
 
+  remarshal = (callPackage ../development/tools/remarshal { }).bin;
```

Comment:
Ok, i did that, but it is kind of ugly in this cases bin should be default output.